command/bridge/common/params.go

```diff
@@ -143,8 +143,11 @@ func (bp *ERC1155BridgeParams) Validate() error {
 }
 
 // ExtractExitEventID tries to extract exit event id from provided receipt
-func ExtractExitEventID(receipt *ethgo.Receipt) (*big.Int, error) {
+func ExtractExitEventID(receipt *ethgo.Receipt) ([]*big.Int, error) {
```

Comment:
Rename it and update the comment accordingly

```suggestion
func ExtractExitEventIDs(receipt *ethgo.Receipt) ([]*big.Int, error) {
```